targets/DEV1-TMU/main.cpp

```diff
@@ -76,7 +76,15 @@ extern "C" void CONmtHbConsChange(CO_NMT* nmt, uint8_t nodeId, CO_MODE mode) {}
 
 extern "C" int16_t COParaDefault(CO_PARA* pg) { return 0; }
 
-extern "C" void COPdoTransmit(CO_IF_FRM* frm) {}
+extern "C" void COPdoTransmit(CO_IF_FRM* frm) {
+    log::LOGGER.log(log::Logger::LogLevel::DEBUG, "Sending PDO as 0x%X with length %d and data: ", frm->Identifier, frm->DLC);
```

Comment:
I'm sure this was useful for debugging, but can you revert this back now that you've confirmed things are working?